vnext/JSI.Desktop.UnitTests/ChakraCoreRuntimeUnitTests.cpp

```diff
@@ -0,0 +1,49 @@
+#include "JsiRuntimeUnitTests.h"
+
+#include "Chakra/ChakraJsiRuntimeFactory.h"
+#include "Chakra/ChakraJsiRuntimeArgs.h"
+#include "MemoryTracker.h"
+
+// TODO (yicyao): Introduces a vcxitem for shared test code and move this there.
```

Comment:
>ODO (y [](start = 4, length = 6)

please add work item id for this

Reply:
Done.

---
In reply to: [300190833](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r300190833) [](ancestors = 300190833)